numpyro/contrib/tfp/distributions.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from tensorflow_probability.substrates.jax import distributions as tfd
+
+import numpyro.distributions as numpyro_dist
+from numpyro.distributions import Distribution as NumPyroDistribution
+from numpyro.distributions import constraints
+
+
+class TFPDistributionMixin(NumPyroDistribution):
+    support = constraints.real
+
+    def __call__(self, *args, **kwargs):
+        key = kwargs.pop('rng_key')
+        return self.sample(*args, seed=key, **kwargs)
+
+
+__all__ = []
+for _name, _Dist in tfd.__dict__.items():
+    if not isinstance(_Dist, type):
+        continue
+    if not issubclass(_Dist, tfd.Distribution):
+        continue
+    if _Dist is tfd.Distribution:
+        continue
+
+    try:
+        _PyroDist = locals()[_name]
```

Comment:
I don't think we should need this since this module doesn't have any distribution in the `locals()` scope.

Reply:
That's right! But we will add some wrappers soon. :D